### PR TITLE
Drop `var` keyword in parameter list

### PR DESCRIPTION
**Pull request by soc**
As discussed earlier!

### CODE REVIEW

File: dora/src/error/msg.rs [hunk 1]
```diff
@@ -485,12 +485,16 @@ impl SemErrorAndPos {
...
 
     pub fn message(&self, vm: &VM) -> String {
-        let file = vm.file(self.file);
-        format!(
-            "error in {} at {}: {}",
-            file.name,
-            self.pos,
-            self.msg.message()
-        )
+        let file = self.filename(vm, self.file);
```
Comment (dinfuehr) [inline dora/src/error/msg.rs:488]: Thanks for tackling this! It was indeed very annoying. However I don't think this is the right way to fix this. I have a commit here that should fix this problem: https://github.com/dinfuehr/dora/commit/3e7ccbf364b3c7514261387595ec2c00104a3931

File: bench/binarytrees/binarytrees.dora [hunk 2]
```diff
@@ -44,7 +44,8 @@ fun loops(iterations: Int, depth: Int) {
     depth.toString() + "\t check: " + check.toString());
 }
 
-fun createTree(var depth: Int) -> TreeNode {
+fun createTree(dep: Int) -> TreeNode {
+  var depth = dep;
```
Comment (dinfuehr) [inline bench/binarytrees/binarytrees.dora:48]: For all those replacements I would prefer if we would just have: `var depth = depth`, so not renaming the variable.

File: dora/src/error/msg.rs (hunk 1 above)
Comment (soc) [reply dora/src/error/msg.rs:488]: Great, thanks, will drop this!

File: bench/binarytrees/binarytrees.dora (hunk 2 above)
Comment (soc) [reply bench/binarytrees/binarytrees.dora:48]: Ah, right, completely forgot we can shadow like this. That's of course much better, will do it!

Comment (soc) [reply bench/binarytrees/binarytrees.dora]: Done!

**Comment by soc** [comment]
@dinfuehr Oh, the conflicts are new. :-( Will fix them.

**Comment by dinfuehr** [comment]
Thanks for pinging me! Looks good, thanks!

**Comment by soc** [comment]
Thanks!

**Pull request merged**